### ISSUE TITLE
Empty line in empty blocks

### ISSUE THREAD

**Issue by Arnoider** [codecadwallader/codemaid#431]
Is there any way to config CodeMaid to keep/enforce a single empty line in empty blocks? Instead of this:

```
int Method()
{
}
```

I would like to have

```
int Method()
{

}
```

I am just used to it and I find it more "readable", but I did not find any "insert black line before/after" option that would satisfy my needs.

**Comment by codecadwallader** [comment]
There isn't currently a way to insert a blank line before/after closing braces, but there are options to remove them which you may want to turn off if you don't want CodeMaid to take them away.  Are there other scenarios where you put in blank lines between braces?  As an example if that was an if statement, would you still inject a blank line?

**Comment by Arnoider** [comment]
Yes, I would. Actually I can not find any example of an empty block where I do not use a blank line. The purpose is to emphasize/stress the fact, that it is empty. In case of if statement it is not so important to me, because we have got also the rule, that the empty block in if statement must contain a comment at least. :-)

Btw. CodeMaid is great. I love it for its configuration capabilities. Thanks!

**Comment by codecadwallader** [comment]
Makes sense, thanks for the information.  Very glad to hear you like CodeMaid. :)
